addons/2d-color-picker/addon.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "2D Color Picker",
+  "description": "Replaces saturation and brightness sliders with a 2d color picker",
```

Comment:
Name and description TBD

Reply:
fixed. the name is now correctly formatted and the description has a period at the end.